jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -289,6 +294,16 @@ public BundleJson apply(final SubscriptionBundle input) {
         return Response.status(Status.OK).entity(result).build();
     }
 
+    private List<SubscriptionBundle> filterBundles(final List<SubscriptionBundle> subscriptionBundlesForAccountId, final List<String> bundlesFilter) {
+        List<SubscriptionBundle> result = new ArrayList<SubscriptionBundle>();
+        for (SubscriptionBundle subscription : subscriptionBundlesForAccountId) {
+            if (bundlesFilter.contains(subscription.getId().toString())) {
```

Comment:
Variable name should probably be change to `bundle` or `subscriptionBundle` since this is not a subscription object.